projects/common/src/navigation/navigation.service.ts

```diff
@@ -60,6 +61,53 @@ export class NavigationService {
     return this.currentParamMap.getAll(parameterName);
   }
 
+  public navigateToUrl(url: string): Observable<boolean> {
+    const navigationParams = this.buildNavigationParams(url)!;
+
+    return from(this.router.navigate([navigationParams[0]], navigationParams[1]));
+  }
+
+  public buildNavigationParams(url: string): [NavigationPath, NavigationExtras] | undefined {
```

Comment:
should never return undefined IMO - I'd call this an error situation.

Reply:
Also, detailed above, I think returning a tuple publicly is a confusing design - would love to condense down to an object.

Reply:
okay.